HIPs/hip-1.md

```diff
@@ -0,0 +1,271 @@
+---
+hip: 1
+title: HIP Purpose and Guidelines
+status: Active
+type: Meta
+author: Steven McKie <mckie@amentum.org>
+created: 2020-08-07
+updated: 2020-08-07
+---
+
+## What is an HIP?
+
+HIP stands for Handshake Improvement Proposal. An HIP is a design document providing information to the Handshake community, or describing a new feature for Handshake or its processes or environment. The HIP should provide a concise technical specification of the feature and a rationale for the feature. The HIP author is responsible for building consensus within the community and documenting dissenting opinions.
+
+## HIP Rationale
+
+We intend HIPs to be the primary mechanisms for proposing new features, for collecting community technical input on an issue, and for documenting the design decisions that have gone into Handshake Because the HIPs are maintained as text files in a versioned repository, their revision history is the historical record of the feature proposal.
+
+For Handshake client implementers, HIPs are a convenient way to track the progress of their implementation. Ideally each implementation maintainer would list the HIPs that they have implemented. This will give end users a convenient way to know the current status of a given implementation or library.
+
+## HIP Types
+
+There are three types of HIP:
+
+- A **Standard Track HIP** describes any change that affects most or all Handshake implementations, such as a change to the network protocol, a change in block or transaction validity rules, proposed naming application standards/conventions, or any change or addition that affects the interoperability of applications using Handshake. Furthermore Standard HIPs can be broken down into the following categories. Standards Track HIPs consist of three parts, a design document, implementation, and finally if warranted an update to the [formal specification].
```

Comment:
Again since many HIPs won't necessarily apply to consensus (like name swaps or standards for selling a name) I think we can either add a new section "Application" or rephrase this to be more lenient around "all implementations". We have this whole DNS resolver layer as well that may not be critical for all HNS resolvers.